code/modules/surgery/organs/heart.dm

```diff
@@ -84,6 +84,8 @@
 		to_chat(owner, "<span class='userdanger'>Your heart has been replaced with a cursed one, you have to pump this one manually otherwise you'll die!</span>")
 		RegisterSignal(owner, COMSIG_LIVING_PRE_DEFIB, PROC_REF(just_before_revive))
 		RegisterSignal(owner, COMSIG_LIVING_DEFIBBED, PROC_REF(on_defib_revive))
+		in_grace_period = TRUE
+		addtimer(VARSET_CALLBACK(src, in_grace_period, FALSE), 10 SECONDS)
```

Comment:
10 SECONDS is the same duration the revival_grace_period var has, do you think they should both be related to it or kept separated like that?

Reply:
Yeah, might as well just add a define for this

Reply:
uses the var now